index.html

```diff
@@ -76,6 +76,11 @@ <h2>Plus UI</h2>
           <i class="fa fa fa-picture-o nav-icon" aria-hidden="true"></i><span class="nav-name">Responsive Images</span>
         </div>
       </a>
+      <a class="nav__item" href="#modal">
+        <div>
+          <i class="fa fa-id-card-o nav-icon" aria-hidden="true"></i><span class="nav-name">Modal</span>
```

Comment:
suggestion: in line no. 79 you use "__" and in line no. 81 you use  "-" in class names, try to be consistent in one method